README.md

```diff
@@ -1,14 +1,14 @@
-This is a pkcs11 soft token written in rust
+This is a pkcs11 soft token written in rust.
 
 # Dependencies
 
  * rustc
  * openssl dependencies
  * sqlite
 
-Note, the default feature links against the system installed OpenSSL
-libraries, you need the OpenSSL development packages to build with
-the default features selection.
+Note, the default feature links against the system installed openssl
+and does not use the openssl submodule, you need the openssl
+development package to build with the default features.
```

Comment:
this is the last part I am not sure about. This was substantially reworded with the removal of the openssl submodule in #213 and looks like your changes revert most of it back, such as refernces to the submodule, which is gone now.

Reply:
Whoops, I tried to drop the changes that were redundant, but I missed this one.  Thanks for double checking.